demo/sub/deep/d.md

```diff
@@ -4,7 +4,7 @@
 * [Chapter 2a](#/2)
 * [Chapter 2b](#/2/1)
 * [Chapter 3](#/3)
-* [Entirely different deck](../../a.md)
+* [Entirely different deck](../../../a.md)
```

Comment:
I think it is correct with `../..`, as there are two folders between the files. Maybe in a specific setup it fails, but `reveal-md ./demo/` works fine.